common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationFragment.java

```diff
@@ -104,20 +104,34 @@ public void onCreate(@Nullable final Bundle savedInstanceState) {
     }
 
     void finish() {
+        final String methodName = "#finish";
```

Comment:
For consistency, we updated the tags to use a colon to define the `methodName`.
Moreso, we can have:
```java
final String methodTag = TAG +":finish";
```
in case the tag is needed in multiple locations.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1701